plugins/hlf.yaml

```diff
@@ -19,20 +18,36 @@ spec:
         matchLabels:
           os: darwin
           arch: amd64
-      uri: https://github.com/kfsoftware/hlf-operator/releases/download/v1.1.0/kubectl-hlf_darwin_amd64.zip
-      sha256: 7d99eb3cd754e30db82529f62d76d840ea28cfd99999d67bdd4624edfd5d5738
+      uri: https://github.com/kfsoftware/hlf-operator/releases/download/v1.2.1/kubectl-hlf_1.2.1_darwin_amd64.zip
+      sha256: 83126ad70f3c45274479f038029e16acd9c6aa4765d0e84145f33ba8957164a8
+      files:
+        - from: kubectl-hlf_1.2.1_darwin_amd64
+          to: kubectl-hlf
+        - from: ./LICENSE
+          to: .
       bin: kubectl-hlf
+
     - selector:
         matchLabels:
           os: linux
           arch: amd64
-      uri: https://github.com/kfsoftware/hlf-operator/releases/download/v1.1.0/kubectl-hlf_linux_amd64.zip
-      sha256: f586be1e5c4de89576c8953b65af61be4a6defa1b4d0ede29d31b2bd823b4647
+      uri: https://github.com/kfsoftware/hlf-operator/releases/download/v1.2.1/kubectl-hlf_1.2.1_linux_amd64.zip
+      sha256: 80ff54d8014ec884be444afc471e05f43abb7fe50580ac07cd1f768cfd457455
+      files:
+        - from: kubectl-hlf_1.2.1_linux_amd64
```

Comment:
This file name will cause auto-approvals to fail at the next version. No other plugin's binary names in the archive contain version strings.

Why do you list files explicitly here? Are there other stuff in the archive?

Reply:
If I remove the files will it work?

Reply:
I have removed the files section and re-uploaded the zips with the right binary names.